source/addons-and-dependencies/managing-dependencies.md

```diff
@@ -33,12 +33,25 @@ without the need to `import` them.
 Provide the asset path as the first and only argument.
 
 ```ember-cli-build.js
-app.import('bower_components/moment/moment.js');
+app.import('node_modules/moment/moment.js');
 ```
 
 You will need to add `"moment"` to the `globals` section in `.eslintrc.js` to prevent ESLint errors
 about using an undefined variable.
 
+Alternatively, you can also use the amd transform with libraries that support both global and amd, to allow you to 
```

Comment:
"global and amd, allowing you to..."

Reply:
I don't understand this sentence. `transformation: 'amd'` allows you to transform anonymous AMD to named AMD afaik, but I wouldn't understand that from this description.

Reply:
@Turbo87 Yeah, I was trying to reference stuff like this:

```js
;(function (global, factory) {
    typeof exports === 'object' && typeof module !== 'undefined' ? module.exports = factory() :
    typeof define === 'function' && define.amd ? define(factory) :
    global.moment = factory()
}(this, (function () { 'use strict';
```

I'm not great at the terminology involved in explaining this, but I wanted it to be explained that libraries that do these checks support global or named AMD, etc. I would definitely love some help crating this section with clear and correct terminology 😃